Area51/Sources/TableViewCells/ListingTableViewCell.swift

```diff
@@ -1,12 +1,42 @@
-import ListingService
 import UIKit
+import ImageService
+import ListingService
 
 class ListingTableViewCell: UITableViewCell {
-    @IBOutlet var titleLabel: UILabel!
+    @IBOutlet private var titleLabel: UILabel!
+    @IBOutlet private var bodyLabel: UILabel!
+    @IBOutlet private var thumbnailImageView: NetworkImageView!
+
+    static var reuseIdentifier: String {
+        return "ListingTableViewCell"
+    }
 }
 
 extension ListingTableViewCell: ListingDisplayable {
-    func display(listing: Listing) {
-        self.titleLabel.text = listing.title
+    func display(_ listing: Listing) {
+        configureTitleText(with: listing)
+        configureThumbnailImageView(with: listing)
+        configureBodyText(with: listing)
+    }
+}
+
+private extension ListingTableViewCell {
+    func configureTitleText(with listing: Listing) {
+        titleLabel.text = listing.title.stripHtml
```

Comment:
In a later PR, we can create ViewModels which already hold the preprocessed data.

Reply:
100% agree with this. I generally use the MVVM pattern with my projects. I'll create an ice for this and solve it in a upcoming PR